README.rst

```diff
@@ -73,45 +93,77 @@ Update your ``.travis.yml`` file with the following::
       - export DISPLAY=:99.0
       - sh -e /etc/init.d/xvfb start
 
-Testing in Travis CI with multiple versions of Plone
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``travis.cfg`` file in your package should look like this::
+Testing in Travis CI with multiple versions of Plone and Python
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``buildout.cfg`` file in your package should look like this::
 
     [buildout]
     extends =
-        https://raw.github.com/collective/buildout.plonetest/master/travis-4.x.cfg
+        https://raw.github.com/collective/buildout.plonetest/master/test-5.x.cfg
 
     package-name = collective.foo
     package-extras = [test]
 
-    allow-hosts +=
-        code.google.com
-        robotframework.googlecode.com
+    [versions]
+    setuptools = 41.2.0
+    zc.buildout = 2.13.2
+
+The versions are optional, but they can help in avoiding restarts when buildout tries to upgrade itself to a version pinned in Plone.
+It is fine to start without them, but when you run into problems in `Travis CI`_, consider adding them.
+In this way, all Plone versions use the same buildout and setuptools versions.
```

Comment:
Shouldn't each version of Plone have its own version of zc.buildout/setuptools?

see:

https://github.com/plone/bobtemplates.plone/blob/4a792eb0899317e406310661eee5b55d40a5d915/bobtemplates/plone/addon/.travis.yml.bob#L25

Reply:
They can, but they don't have to. Latest buildout and setuptools work fine for all current versions of Plone. With the same version, we avoid the need for extra files.